docs/workflows/common-workflow-steps/appshield-scanner.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Appshield Scanner for Android/iOS
+description: Appshield Scanner performs dynamic runtime scans/analysis and AI supported static (file-based) inspections for mobile app files (APK, AAB, IPA) to detect existing security mechanisms and indicates whether an app is secure or not.
+tags: [android, ios, mobile, security, scan, appshield-scanner]
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Appshield App Scanner for Android / iOS
+
+Appshield Scanner starts its analysis by accepting the application file (AAB/APK for Android, IPA for iOS) and starting dynamic runtime tests after initial file and format validations. 
+It is important to note that while many scanning solutions use ARM/x86_64 based emulators and sandbox environments, Appshield performs all these dynamic tests in **real/physical Android and/or iOS devices.**
+At the end of the dynamic test session, Appshield indicates which security measures/hardenings are present and implemented in the app. If some testcases were failed due to unforeseen errors or cannot be tested various bypass mechanisms 
+implemented by the app itself, Appshield then starts a static, AI-powered analysis for mentioned testcases to gather additional findings and then reaches a final verdict.
```

Comment:
1. Unnecessary newline:
A new line was started before the sentence was completed.
2. Article usage:
“for mentioned testcases” → should be “for the mentioned test cases”